src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx

```diff
@@ -47,29 +47,30 @@ Follow these steps.
     <CollapserGroup>
     <Collapser
         id="step1"
-        title="Step 1. Define your signal"
+        title="Step 1. Add alerts"
```

Comment:
@Fallenstedt I think the docs should match the UI, so the step titles should be:
1. Define your signal
2. Adjust to signal behavior
3. Define thresholds
4. Add details. 

Then, within each title, we can give more information/explanation that the UI doesn't have. What do you think?

Reply:
@Fallenstedt In the UI, there's an option to select "Golden signal/metric". Is there a reason why this option isn't described in this section? If it was just an oversight, we'll need to provide info on the benefits/use cases of using a NRQL query vs. the golden signal/metric.

Reply:
There are two separate modes for the condition builder creation. You use "guided" mode or "query" mode. 

Since these docs are focused on creating a an alert condition from a NRQL query, then we are using "query" mode. 

These are the new steps for query mode. Maybe we can consolidate it three steps for this doc. 

1. Tell us where to look
2. Set thresholds
3. Add details. 

We can then tell them they need to click "write your own query" at the fig caption

Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts & AI > Alert Conditions & Policies > + New alert condition**. **Then click "Write your own query"**


A video of the new experience. 

https://github.com/newrelic/docs-website/assets/13971824/ae48906e-69a7-4397-8eba-d592999e2424

Reply:
@ally-sassman 
> In the UI, there's an option to select "Golden signal/metric". Is there a reason why this option isn't described in this section? If it was just an oversight, we'll need to provide info on the benefits/use cases of using a NRQL query vs. the golden signal/metric.

This was not part of our original documentation update we made (#tmp-nrql-alert-condition-docs) https://github.com/newrelic/docs-website/pull/12149 

We _can_ include a section on how to use "guided" mode. The steps have some extra information and screens in "guided" mode

The steps are 
1. Tell us where to look 
2. Tell us what to watch
3. Set thresholds
4. Add details

The first two steps are not building a query. Instead, they are helping you choose an entities in you account. Once you have chosen entities in your account, then you can select those entities' metrics to create a NRQL alert condition. 

If you'd like to meet 1 - 1 so I can show you this workflow, please let me know.